hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/app/templates/components/yarn-queue-partition-capacity-labels.hbs

```diff
@@ -16,59 +16,255 @@
-    <span class="label-key">absolute max capacity</span>
+  </li>
+  {{!
+  /scheduler/schedulerInfo/queues/queue[2]/absoluteMaxCapacity[text()='100.0']
+  }}
+  <li class="yarn-list">
+    <span class="label-key" title="Absolute Configured Max Capacity: percentage referring to the configured maximum resources from the total available resources for the partition. This value will be different from the configured max capacity only when the parent queue is not the root queue.">absolute configured max capacity:</span>
     <span class="label-value">{{data.absoluteMaxCapacity}}%</span>
-  </span>
-</div>
-{{#if data.isWeightMode}}
-<div class="top-1">
-  <span class="yarn-label secondary">
-    <span class="label-key">weight</span>
-    <span class="label-value">{{data.weight}}</span>
-  </span>
-  <span class="yarn-label secondary">
-    <span class="label-key">normalized weight</span>
-    <span class="label-value">{{data.normalizedWeight}}</span>
-  </span>
-</div>
-{{else}}
-<div class="top-1">
-  <span class="yarn-label secondary">
-    <span class="label-key">configured capacity</span>
-    <span class="label-value">{{data.capacity}}%</span>
-  </span>
-  <span class="yarn-label secondary">
-    <span class="label-key">configured max capacity</span>
-    <span class="label-value">{{data.maxCapacity}}%</span>
-  </span>
-</div>
-{{/if}}
+  </li>
+
+
+  {{#if data.isWeightMode}}
+
+    <li class="yarn-list">
+      <span class="label-key" title="Configured Weight">weight:</span>
+      <span class="label-value">{{data.weight}}</span>
+    </li>
+    <li class="yarn-list">
+      <span class="label-key" title="Normalized Weight">normalized weight:</span>
+      <span class="label-value">{{data.normalizedWeight}}</span>
+    </li>
+
+  {{else}}
+
+    {{!
+    /scheduler/schedulerInfo/queues/queue[2]/capacities/queueCapacitiesByPartition[2]/capacity[text()='85.44922']
+    }}
+    <li class="yarn-list">
+      <span class="label-key" title="Configured Capacity: percentage = 'configured resources of the queue' / 'total available resources for the partition' * 100">configured capacity:</span>
+      <span class="label-value">{{data.capacity}}%</span>
+    </li>
+
+    {{!
+    /scheduler/schedulerInfo/queues/queue[2]/maxCapacity[text()='100.0']
+    }}
+    <li class="yarn-list">
+      <span class="label-key" title="Configured Max Capacity: percentage = 'configured maximum resources of the queue' / 'total available resources for the partition' * 100">configured max capacity:</span>
+      <span class="label-value">{{data.maxCapacity}}%</span>
+    </li>
+
+    {{!
+    /scheduler/schedulerInfo/queues/queue[2]/capacities/queueCapacitiesByPartition/configuredMinResource/memory[text()='54060']
+    /scheduler/schedulerInfo/queues/queue[2]/capacities/queueCapacitiesByPartition/configuredMinResource/vCores[text()='5']
+    }}
+    <li class="yarn-list">
+      <span class="label-key" title="Configured min capacity">configured min capacity:</span>
```

Comment:
Just to avoid complicating things even more, can you please use the same naming for these parameters as they are on UI1? Attaching a screenshot for help. In this case `configured min capacity` would become `Configured Capacity`. `used am resources` would be `Used Application Master Resources`. It's longer, but for the users it's easier to match the information they had on the old UI.


<img width="884" alt="Screenshot 2025-02-11 at 16 33 50" src="https://github.com/user-attachments/assets/b65c07e3-4504-4cdf-a93c-df3f96c5dbbd" />